### PR TITLE
Distinguish sandbox/initContainer/container in runtime

### PR DESCRIPTION
**Pull request by kkkkun**
#### What type of PR is this?
/kind flake

#### What this PR does / why we need it:

It doesn't distinguish initContainer and container. It's difficult when we want to do some operations of initContainer in runtime, so we want to  add labels to distinguish.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/kubelet/kuberuntime/labels.go [hunk 1]
```diff
@@ -104,6 +105,17 @@ func newContainerLabels(container *v1.Container, pod *v1.Pod) map[string]string
...
+	for _, c := range pod.Spec.InitContainers {
+		if c.Name == container.Name {
+			labels[types.KubernetesContainerTypeLabel] = types.PodInitContainer
+		}
+	}
+	for _, c := range pod.Spec.Containers {
+		if c.Name == container.Name {
+			labels[types.KubernetesContainerTypeLabel] = types.PodContainer
+		}
+	}
```
Comment (bart0sh) [inline pkg/kubelet/kuberuntime/labels.go:117]: @kkkkun Do we need two loops here? If container type is either types.PodInitContainer or types.PodContainer we can omit one of the loops, I guess.

Comment (kkkkun) [reply pkg/kubelet/kuberuntime/labels.go:113]: Thanks & fixed

**Comment by bart0sh** [comment]
@kkkkun 
> we want to do some operations of initContainer in runtime

Can you give some examples of such operations?

**Pull request closed without merging**